easylist/easylist_adservers.txt

```diff
@@ -5517,6 +5517,7 @@
 ||funtoday.info^
 ||fupzhtvo.com^
 ||fuqxpjxay.com^
+||furlsstealbilk.com^
```

Comment:
`.com/1clkn/` would be more optimal...

![image](https://user-images.githubusercontent.com/44526987/147083986-a79989f5-478f-4dc7-9ee4-09a8452093b8.png)